agent/src/agent/pipeline/config/jython_scripts/test_pipelines/zabbix.py

```diff
@@ -0,0 +1,33 @@
+global sdc
+
+try:
+    sdc.importLock()
+    import sys
+    import os
+    sys.path.append(os.path.join(os.environ['SDC_DIST'], 'python-libs'))
+    import requests
+    import traceback
+finally:
+    sdc.importUnlock()
+
+
+try:
+    res = requests.post(
+        sdc.userParams['URL'] + '/api_jsonrpc.php',
+        json={
+            'jsonrpc': '2.0',
+            'method': 'user.login',
+            'params': {'user': sdc.userParams['USER'],
+                       'password': sdc.userParams['PASSWORD']},
+            'id': 1,
+            'auth': None,
+        },
+        verify=sdc.userParams['VERIFY_SSL']
+    )
```

Comment:
add timeout parameter to the request [https://docs.python-requests.org/en/latest/user/quickstart/#timeouts](https://docs.python-requests.org/en/latest/user/quickstart/#timeouts)

same for promql and influx/influx2

Reply:
Added REQUEST_TIMEOUT parameter for test_pipelines. Set to 10 sec